README.md

```diff
@@ -53,10 +57,44 @@ cp .env.example .env
 The `.env` file contains the following values:
 
 ```bash
-SERVICE_URL="" # (Required) The endpoint where events will be send to
-APPLICATION_NAME="" # (Optional) the application identier. This is useful when you have more than one application sending events
+SERVICE_URL=""
+APPLICATION_NAME=""
+PERFORMANCE_OBSERVER_METRICS=""
 ```
 
+### Service url (mandatory)
+
+The url where events will be send to
+
+### Application name (optional)
+
+This is app's identifier and is useful when you have more than one application which is sending events -
+
+```bash
+APPLICATION_NAME="my-app-name"
+```
```

Comment:
Why did you include another code sample here, but not for the others?

Reply:
I'm cleaning up from the previous readme